proto/babylon/zoneconcierge/v1/packet.proto

```diff
@@ -18,8 +18,7 @@ message ZoneconciergePacketData {
   oneof packet {
     BTCTimestamp btc_timestamp = 1;
     babylon.btcstaking.v1.BTCStakingIBCPacket btc_staking = 2;
-    ConsumerRegisterIBCPacket consumer_register = 3;
```

Comment:
Bye nice automatic consumer registration.🙂

Reply:
Yeah somehow this turns out to be much more complicated -- both Babylon and consumer have to pass gov prop for integration, and one cannot simply deploy a chain that integrates with Babylon without any approval

Reply:
On second thought, maybe a bit late for this but, what about falling back to / keeping automatic registration for Cosmos chains, in the case of permissionless integration?